query-node/schema.graphql

```diff
@@ -65,55 +52,40 @@ type ChannelCategory @entity {
 
   channels: [Channel!] @derivedFrom(field: "category")
 
-  happenedIn: Block!
+  happenedIn: BigInt!
 }
 
-"Storage asset"
-union Asset = AssetUrl | AssetStorage
+"Asset availability representation"
+enum AssetAvailability {
+  ACCEPTED, # asset is available in storage
+  PENDING, # asset is being uploaded to storage
+  URL, # asset is referencing URL
+  INVALID, # invalid storage (meta)data used
+}
+
+"Asset representation"
+union Asset = AssetUrl | AssetStorage | AssetDeleted
```

Comment:
I thought we were dropping `AssetDeleted`?

Reply:
Yes, it was mistakenly kept there. Now the whole structure is simplified and flatten.